app/modules/u8g.c

```diff
@@ -820,6 +820,31 @@ static void lu8g_digital_write( u8g_t *u8g, uint8_t pin_index, uint8_t value )
         platform_gpio_write( pin, value );
 }
 
+void u8g_Delay(u8g_t *u8g, uint16_t msec)
+{
+    const uint16_t chunk = 50;
+
+    if (u8g->use_delay == 0)
+        return;
+
+    while (msec > chunk)
+    {
+        os_delay_us( chunk*1000 );
+        msec -= chunk;
+    }
+    if (msec > 0)
+        os_delay_us( msec*1000 );
+}
+void u8g_MicroDelay(void)
```

Comment:
Formal reasons - The port needs to provide a function to u8glib. Introducing this as a macro would move implementation specific stuff to the generic header files.